server/service/integration_core_test.go

```diff
@@ -2404,16 +2404,22 @@ func (s *integrationTestSuite) TestGlobalPoliciesProprietary() {
 	assert.Equal(t, "admin1@example.com", gpResp.Policy.AuthorEmail)
 	assert.Equal(t, "darwin", gpResp.Policy.Platform)
 
-	mgpParams := modifyGlobalPolicyRequest{
-		ModifyPolicyPayload: fleet.ModifyPolicyPayload{
-			Name:        ptr.String("TestQuery4"),
-			Query:       ptr.String("select * from osquery_info;"),
-			Description: ptr.String("Some description updated"),
-			Resolution:  ptr.String("some global resolution updated"),
-		},
-	}
-	mgpResp := modifyGlobalPolicyResponse{}
-	s.DoJSON("PATCH", fmt.Sprintf("/api/latest/fleet/policies/%d", gpResp.Policy.ID), mgpParams, http.StatusOK, &mgpResp)
+	response := s.DoRaw("PATCH", fmt.Sprintf("/api/latest/fleet/policies/%d", gpResp.Policy.ID), []byte(`{
+		"PolicyID": 0,
+		"name": "TestQuery4",
+		"query": "select * from osquery_info;",
+		"description": "Some description updated",
+		"resolution": "some global resolution updated",
+		"platform": null,
```

Comment:
Do we need the extra nulls here and elsewhere?

Reply:
I don't believe so. But this is what the structs serialized to so I left them in there in case the endpoint did something weird with unsetting those properties. They kinda clutter the test, so I can remove them. No strong opinions here.

Reply:
Let's drop 'em and see what happens. I'd expect these endpoints to obey PATCH semantics, so skipping values should result in no change for those values.

Reply:
I would actually presume that setting them to `null` would unset them given the way patch works. So this feels more correct to me. 👍